client/src/components/main/coffee_shop/CoffeeShopIndex.scss

```diff
@@ -1,10 +1,19 @@
+#panel-coffee {
```

Comment:
This should be a class grabber because `id`'s should only refer to a single DOM element